src/EventSubscriber/TideSiteGetRouteSubscriber.php

```diff
@@ -6,6 +6,8 @@
 use Drupal\Core\StringTranslation\StringTranslationTrait;
 use Drupal\tide_api\Event\GetRouteEvent;
 use Drupal\tide_api\TideApiEvents;
+use Drupal\tide_api\TideApiHelper;
```

Comment:
@sonnykt @anthony-malkoun `tide_api` isn't set as a dependency of `tide_site` in `composer.json` but it is being used as if it were.

They've always been dependencies of one another  (not necessarily defined in code) but I'm interested in discovering if it's desired.

I thought we wanted to decouple `tide_api` and `tide_site` but is there ever going to be a scenario of `tide_site` being used without `tide_api`?

If `tide_api` is going to be a dependency it should be defined as one in the `composer.json` file.

Reply:
@GROwen tide_site has support of tide_api, but tide_api is not a dependency of tide_site. This class is only loaded if tide_api is enabled: https://github.com/dpc-sdp/tide_site/pull/25/files#diff-ce88537285ddcf6167a10cefdfb5a523R48

Reply:
the scenarios you mentioned are covered in behat tests of both tide_api and tide_site.